sklearn-demo.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>Hmm, unfortunately it's not possible to tell you to only delete this <code># keep two dimensions two plot</code>  comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>right, I also found it a bit coarse-grained that comments are per cell.</p>